.github/workflows/create-release.yml

```
@@ -0,0 +1,18 @@
+name: Create release
+
+on:
+  push:
+    tags:
+    - '*'
```

Basically, we've used `vX.Y.Z` format as a version tag. How about this?
```suggestion
    - "v*.*.*"
```

Note: This was described at here: https://github.com/softprops/action-gh-release#-limit-releases-to-pushes-to-tags